shell/browser/api/electron_api_power_monitor_mac.mm

```diff
@@ -45,6 +60,18 @@ - (void)addEmitter:(electron::api::PowerMonitor*)monitor_ {
   self->emitters.push_back(monitor_);
 }
 
+- (void)isSuspending:(NSNotification*)notify {
+  for (auto*& emitter : self->emitters) {
```

Comment:
This was not introduced in this PR, but there is no advantage to taking a reference to the pointer here.

`for (auto*& emitter : self->emitters)` => `for (auto* emitter : self->emitters)`

same for all below.

Reply:
@deepak1556 this was done to fix chromium style errors: https://github.com/electron/electron/commit/cdc502230561d7e67c019034a6ffce9d7d748c40

Reply:
Yes that was because we were not specifying the type for raw pointer.

`for (auto* emitter : self->emitters)` // valid
`for (auto emitter : self->emitters)` // not valid